### PR TITLE
change az cli-extension output-folder

### PR DESCRIPTION
**Pull request by qiaozha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,7 @@ See documentation [here](doc/00-overview.md)
 
 python:
     reason: 'make sure python flag exists to load config in python.md'
-debug-output-folder: $(output-folder)/_az_debug
-az-output-folder: $(output-folder)
+debug-output-folder: $(az-output-folder)/_az_debug
```
Comment (RodgeFu) [inline README.md:9]: >az-output-folder [](start = 23, length = 16)

the az-output-folder is empty in this case, isn't it?

Comment (qiaozha) [reply README.md:9]: no autorest will read the configuration from the readme.az.md

Comment (RodgeFu) [reply README.md:9]: i see, then we should set output-folder=az-output-folder, right?

---
In reply to: [407913102](https://github.com/Azure/autorest.az/pull/302#discussion_r407913102) [](ancestors = 407913102)

File: src/test/scenario-test.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ require('source-map-support').install();
 
 @suite class Process {
     async runAz(directory: string, each: string) {
-        let cmd = `${__dirname}/../../` + "node_modules/.bin/autorest --version=3.0.6271 --az --use=" + `${__dirname}/../../` + " " + directory + "/configuration/readme.md --output-folder=" + directory + "/tmpoutput/src/" + each;
+        let cmd = `${__dirname}/../../` + "node_modules/.bin/autorest --version=3.0.6271 --az --use=" + `${__dirname}/../../` + " " + directory + "/configuration/readme.md --azure-cli-extension-folder=" + directory + "/tmpoutput/";
```
Comment (RodgeFu) [inline src/test/scenario-test.ts:13]: >azure-cli-extension-folder [](start = 174, length = 26)

where is azure-cli-extension-folder used? in the code?

File: README.md (hunk 1 above)
Comment (qiaozha) [reply README.md:9]: emmm, you mean don't change the python-sdk-output-folder in readme.az.md?

File: src/test/scenario-test.ts (hunk 2 above)
Comment (qiaozha) [reply src/test/scenario-test.ts:13]: this is the scenarios test command line. and azure-cli-extension-folder is at the right side of this line.

**Pull request merged**